admin_guide/manage_scc.adoc

```diff
@@ -429,56 +429,56 @@ To provide additional capabilities:
 === Modify Cluster Default Behavior
 
 To modify your cluster so that it does not pre-allocate UIDs, allows containers
-to run as any user, and prevents privileged containers:
+to run as any user, but prevents privileged containers, grant access to the
+*anyuid* SCC for everyone:
 
-[NOTE]
-====
-In order to preserve customized SCCs during upgrades, do not edit settings on
-the default SCCs other than priority, users, groups, labels, and annotations.
-====
-
-. Edit the *restricted* SCC:
-+
 ----
- $ oc edit scc restricted
+ $ oc adm policy add-scc-to-group anyuid system:authenticated
 ----
 
-. Change `*runAsUser.Type*` to *RunAsAny*.
-
-. Ensure `*allowPrivilegedContainer*` is set to false.
-
-. Save the changes.
-
 To modify your cluster so that it does not pre-allocate UIDs and does not allow
-containers to run as root:
+containers to run as root, grant access to the *nonroot* SCC for everyone:
 
-. Edit the *restricted* SCC:
-+
 ----
- $ oc edit scc restricted
+ $ oc adm policy add-scc-to-group nonroot system:authenticated
 ----
 
-. Change `*runAsUser.Type*` to *MustRunAsNonRoot*.
-
-. Save the changes.
+[WARNING]
+====
+Be very careful with any modifications that have a cluster-wide impact! When
+SCC is being granted to all authenticated users, as in the example above, it
+will also affect Kubernetes and {product-title} components (a web console, an
+integrated docker registry, etc.) which could lead to their inoperability! The
+same considerations should be taking into account when such SCC is being
+modified (for example, the *restricted* SCC is accessible by everyone). Always
+prefer creating a custom SCC and target it only to specific users or groups. In
+such a case, the impact of the possible breakage will be reduced to a minimum.
```

Comment:
In such a case ... -> This way potential issues will be confined to the affected users/groups and will not impact critical cluster components.